ruby_on_rails/app_initialisation.md

```diff
@@ -11,11 +11,16 @@
 * [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not.
 You can do this with `gem update rails`. Beware of beta versions.
 
-* Start a new Rails project using `rails new [project-name] --database=postgresql --no-skip-test --skip-sprockets --webpack --skip_action_mailbox` where the `project-name` is exactly the one you chose before.
-You may want to choose a different database from Postgres, but most of the time that will be your choice.
+* Start a new Rails project using `rails new [project-name] --database=postgresql --skip-test --skip-action-mailbox` where the `project-name` is exactly the one you chose before.
+
+  * You can use the `--css` flag to preconfigure your new Rails app to use a CSS processor/framework. Available options are: `tailwind, bootstrap, bulma, postcss, sass`. This will automatically install the [cssbundling-rails](https://github.com/rails/cssbundling-rails) and [jsbundling-rails](https://github.com/rails/jsbundling-rails) gems. The app then uses the the [esbuild](https://github.com/evanw/esbuild) JS bundler and minifier by default.
+
+  * Esbuild is generally a good choice, but you can also configure an alternative JS bundler using the `--javascript` flag. You have the choice between using one of the node-based bundlers: `esbuild, rollup, webpack` or you can use `importmap`, which is the default. Note that importmaps cannot be used in conjunction with the `--css` option from above.
+
+  * You may want to choose a different database from Postgres, but most of the time that will be your choice.
 If you do not need a DB you may rethink the fact that you may not need Rails at all :) Take a look at [Sinatra](http://www.sinatrarb.com/) or [Angular](https://angular.io/)
 
-* Check if you've got a `.ruby-version` file. If not, create one and enter the Ruby version.
+* Check if you've got a `.ruby-version` file and create one if not . If  there is already one in the project, be sure to remove the `ruby-` prefix from the version number inside the file, leaving only the number itself.
```

Comment:
Why is this necessary?

Reply:
By default the version is now prefixed with `ruby-` (e.g` ruby-3.1.1`). 

That means that this won't work anymore in the Gemfile:
```ruby
ruby File.read(File.join(__dir__, '.ruby-version')).strip
```

Reply:
`asdf` reads it correctly. I suggest, we add a helper class for such Renuo customization things. What do you think?
![CleanShot 2022-03-28 at 10 34 09](https://user-images.githubusercontent.com/245443/160358925-f09d7970-cd35-41b6-b1f3-a61b97da7f0e.png)

Reply:
I'd like to keep it as asdf compatible as possible :D